gtsam/slam/dataset.h

```diff
@@ -260,6 +277,14 @@ struct SfmData {
   SfmTrack track(size_t idx) const {
     return tracks[idx];
   }
+
+  void add_track(SfmTrack t) {
+    tracks.push_back(t);
+  }
+
+  void add_camera(SfmCamera cam){
```

Comment:
I think this should be a const reference, like `void add_camera(const SfmCamera& cam)`.